LayoutTests/fast/text/glyph-display-lists/glyph-display-list-colr-unshared.html

```diff
@@ -17,8 +17,8 @@
 span { font: 48px Ahem COLR; }
 </style>
 <div id=container>
-    <span id=first>A</span>
-    <span id=second>A</span>
+    <span id=first>B</span>
+    <span id=second>B</span>
```

Comment:
If we keep using 'A', will the display list be shared with your change? In other words, what is the difference between caching a display list for letter 'A' and caching a display list for letter 'B'?